yt_dlp/extractor/iprima.py

```diff
@@ -134,10 +134,19 @@ def _real_extract(self, url):
         ), webpage, 'real id', group='id', default=None)
 
         if not video_id:
-            nuxt_data = self._search_nuxt_data(webpage, video_id, traverse='data')
+            nuxt_data = self._search_nuxt_data(webpage, video_id, traverse='data', fatal=False)
             video_id = traverse_obj(
                 nuxt_data, (..., 'content', 'additionals', 'videoPlayId', {str}), get_all=False)
 
+        if not video_id:
+            nuxt_data = self._parse_json(
+                self._search_regex(
+                    r'(?s)<script[^>]+\bid=["\']__NUXT_DATA__["\'][^>]+>(.+?)</script>',
+                    webpage, 'nuxt data'),
+                'nuxt data')
```

Comment:
This does not work.
```
    raise source.error('global flags not at the start '
re.error: global flags not at the start of the expression at position 3
```
Neither does 
```
nuxt_data = self._search_json(
                r'<script[^>]+\bid=["\']__NUXT_DATA__["\'][^>]*>',
                webpage, 'nuxt data', None, end_pattern=r'</script>')
```
because the default pattern only accepts JSON objects, but what the page contains is an array.

I can (and have done) this:
```
nuxt_data = self._search_json(
    r'<script[^>]+\bid=["\']__NUXT_DATA__["\'][^>]*>',
    webpage, 'nuxt data', None, end_pattern=r'</script>', contains_pattern=r'(?s:.+?)')
```